robottelo/hosts.py

```diff
@@ -674,8 +672,8 @@ def register(
             options['hostgroup-id'] = hostgroup.id
         if packages is not None:
             options['packages'] = '+'.join(packages)
-        if repo is not None:
-            options['repo'] = repo
```

Comment:
Could you take care of other tests using this option as well, so we don't break the other tests? like
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_ansible.py#L474
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_remoteexecution.py#L416

Reply:
Done